feature/received/src/main/java/com/susu/feature/received/ReceivedScreen.kt

```diff
@@ -1,28 +1,218 @@
+                        leftIcon = {
+                            Icon(
+                                painter = painterResource(id = com.susu.core.ui.R.drawable.ic_align),
+                                contentDescription = stringResource(R.string.content_description_align_icon),
+                            )
+                        },
+                        onClick = onClickAlignButton,
+                    )
+                    SusuGhostButton(
+                        color = GhostButtonColor.Black,
+                        style = SmallButtonStyle.height32,
+                        text = stringResource(R.string.word_filter),
+                        leftIcon = {
+                            Icon(
+                                painter = painterResource(id = com.susu.core.ui.R.drawable.ic_filter),
+                                contentDescription = stringResource(R.string.content_description_filter_icon),
+                            )
+                        },
+                        onClick = onClickFilterButton,
+                    )
+                }
+
+                LazyVerticalGrid(
+                    modifier = Modifier
+                        .fillMaxSize(),
+                    columns = GridCells.Fixed(2),
+                    verticalArrangement = Arrangement.spacedBy(SusuTheme.spacing.spacing_xxs),
+                    horizontalArrangement = Arrangement.spacedBy(SusuTheme.spacing.spacing_xxs),
+                ) {
+                    item {
+                        LedgerCard(
+                            ledgerType = "결혼식",
+                            title = "나의 결혼식",
+                            currency = 4335000,
+                            count = 164,
+                            onClick = onClickLedgerCard,
+                        )
+                    }
+
+                    item {
+                        LedgerCard(
+                            ledgerType = "결혼식",
+                            title = "나의 결혼식",
+                            currency = 4335000,
+                            count = 164,
+                            onClick = onClickLedgerCard,
+                        )
+                    }
+
+                    item {
+                        LedgerAddCard(
+                            onClick = onClickLedgerAddCard,
+                        )
+                    }
+                }
+            }
+        }
+
+        Text(
+            modifier = Modifier.align(Alignment.Center),
+            text = stringResource(R.string.received_screen_empty_ledger),
+            style = SusuTheme.typography.text_s,
+            color = Gray50,
+        )
+
+        Box(
+            modifier = Modifier
+                .align(Alignment.BottomEnd)
+                .padding(SusuTheme.spacing.spacing_l),
+        ) {
+            SusuFloatingButton(
+                onClick = onClickFloatingAddButton,
+            )
+        }
```

Comment:
```suggestion
        SusuFloatingButton(
            modifier = Modifier.align(Alignment.BottomEnd).padding(SusuTheme.spacing.spacing_l),
            onClick = onClickFloatingAddButton,
        )
```
요렇게 하면 Box 없이도 디자인 시안이랑 일치하는 것 같아요!

Reply:
오 Box 보기 싫었는디 ... ! 감삼다!